packages/esm-ward-app/src/ward-view-header/ward-metric.scss

```diff
@@ -20,6 +20,6 @@
 }
 
 .skeleton {
-  height: 15px;
-  width: 15px;
+  height: layout.$spacing-05;
+  width: layout.$spacing-05;
```

Comment:
`layout.$spacing-05` === 16px. I've bumped the values up by a pixel because 15px seems odd to me. Hopefully that's ok.